cmd/object-api-utils.go

```diff
@@ -1206,7 +1206,20 @@ func hasSpaceFor(di []*DiskInfo, size int64) (bool, error) {
 	}
 
 	if nDisks < len(di)/2 || nDisks <= 0 {
-		return false, fmt.Errorf("not enough online disks to calculate the available space, expected (%d)/(%d)", (len(di)/2)+1, nDisks)
+		var errs []string
+		for _, disk := range di {
+			if disk == nil {
+				continue
```

Comment:
can we also add an error here ? like `a nil disk (index) is found`

Reply:
A `nil` is an offline disk we should just say that.